libs/sdk-ui-kit/package.json

```diff
@@ -95,7 +95,8 @@
         "tinycolor2": "^1.4.2",
         "ts-invariant": "^0.7.3",
         "tslib": "^2.0.0",
-        "uuid": "^8.3.2"
+        "uuid": "^8.3.2",
+        "custom-event": "^1.0.1"
```

Comment:
packages in package.json should be sorted - custom-event should take a place after copy-to-clipboard :)

Reply:
fixed